### PR TITLE
Evpn vxlan debug dump

### PR DESCRIPTION
**Pull request by dgsudharsan**
#### What I did
Dump utility for EVPN and VxLAN

#### How I did it
Added changes implementing vxlan and evpn tables

#### How to verify it
Added UT to verify the changes.

#### New command output (if the output of a command-line utility has changed)
```
dump state vxlan_tunnel vtep_1336
{
    "vtep_1336": {
        "CONFIG_DB": {
            "keys": [
                {
                    "VXLAN_TUNNEL|vtep_1336": {
                        "src_ip": "10.1.0.32"
                    }
                }
            ],
            "tables_not_found": []
        },
        "APPL_DB": {
            "keys": [
                {
                    "VXLAN_TUNNEL_TABLE:vtep_1336": {
                        "src_ip": "10.1.0.32"
                    }
                }
            ],
            "tables_not_found": []
        },
        "ASIC_DB": {
            "keys": [
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL:oid:0x2a00000000069f": {
                        "SAI_TUNNEL_ATTR_DECAP_MAPPERS": "2:oid:0x2900000000069b,oid:0x2900000000069d",
                        "SAI_TUNNEL_ATTR_ENCAP_MAPPERS": "2:oid:0x2900000000069c,oid:0x2900000000069e",
                        "SAI_TUNNEL_ATTR_ENCAP_SRC_IP": "10.1.0.32",
                        "SAI_TUNNEL_ATTR_PEER_MODE": "SAI_TUNNEL_PEER_MODE_P2MP",
                        "SAI_TUNNEL_ATTR_TYPE": "SAI_TUNNEL_TYPE_VXLAN",
                        "SAI_TUNNEL_ATTR_UNDERLAY_INTERFACE": "oid:0x600000000056a"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x2900000000069c": {
                        "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VLAN_ID_TO_VNI"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x2900000000069e": {
                        "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VIRTUAL_ROUTER_ID_TO_VNI"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x2900000000069b": {
                        "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VNI_TO_VLAN_ID"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x2900000000069d": {
                        "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VNI_TO_VIRTUAL_ROUTER_ID"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_TERM_TABLE_ENTRY:oid:0x2b0000000006a0": {
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_ACTION_TUNNEL_ID": "oid:0x2a00000000069f",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_DST_IP": "10.1.0.32",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_TUNNEL_TYPE": "SAI_TUNNEL_TYPE_VXLAN",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_TYPE": "SAI_TUNNEL_TERM_TABLE_ENTRY_TYPE_P2MP",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_VR_ID": "oid:0x3000000000002"
                    }
                }
            ],
            "tables_not_found": [],
            "vidtorid": {
                "oid:0x2a00000000069f": "oid:0x2a",
                "oid:0x2900000000069c": "oid:0x100000029",
                "oid:0x2900000000069e": "oid:0x300000029",
                "oid:0x2900000000069b": "oid:0x29",
                "oid:0x2900000000069d": "oid:0x200000029",
                "oid:0x2b0000000006a0": "oid:0x420000002b"
            }
        },
        "STATE_DB": {
            "keys": [],
            "tables_not_found": [
                "VXLAN_TUNNEL_TABLE"
            ]
        }
    }
}
```
```
dump state vxlan_tunnel_map all
{
    "vtep_1336|map_1336_Vlan2345": {
        "CONFIG_DB": {
            "keys": [
                {
                    "VXLAN_TUNNEL_MAP|vtep_1336|map_1336_Vlan2345": {
                        "vlan": "Vlan2345",
                        "vni": "1336"
                    }
                }
            ],
            "tables_not_found": []
        },
        "APPL_DB": {
            "keys": [
                {
                    "VXLAN_TUNNEL_MAP_TABLE:vtep_1336:map_1336_Vlan2345": {
                        "vlan": "Vlan2345",
                        "vni": "1336"
                    }
                }
            ],
            "tables_not_found": []
        },
        "ASIC_DB": {
            "keys": [
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP_ENTRY:oid:0x3b0000000006a1": {
                        "SAI_TUNNEL_MAP_ENTRY_ATTR_TUNNEL_MAP": "oid:0x2900000000069b",
                        "SAI_TUNNEL_MAP_ENTRY_ATTR_TUNNEL_MAP_TYPE": "SAI_TUNNEL_MAP_TYPE_VNI_TO_VLAN_ID",
                        "SAI_TUNNEL_MAP_ENTRY_ATTR_VLAN_ID_VALUE": "2345",
                        "SAI_TUNNEL_MAP_ENTRY_ATTR_VNI_ID_KEY": "1336"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x2900000000069b": {
                        "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VNI_TO_VLAN_ID"
                    }
                }
            ],
            "tables_not_found": [],
            "vidtorid": {
                "oid:0x3b0000000006a1": "oid:0x10000003b",
                "oid:0x2900000000069b": "oid:0x29"
            }
        }
    }
}
```
```
dump state evpn 'Vlan2345:11.1.0.32'
{
    "Vlan2345:11.1.0.32": {
        "APPL_DB": {
            "keys": [
                {
                     "VXLAN_REMOTE_VNI_TABLE:Vlan2345:11.1.0.32": {
                         "vni": "1336"
                      }
                }
            ],
            "tables_not_found": []
        },
        "ASIC_DB": {
            "keys": [
                {
                        "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL:oid:0x2a0000000007e1": {
                             "SAI_TUNNEL_ATTR_UNDERLAY_INTERFACE": "oid:0x600000000056a",
                             "SAI_TUNNEL_ATTR_PEER_MODE": "SAI_TUNNEL_PEER_MODE_P2P",
                             "SAI_TUNNEL_ATTR_DECAP_MAPPERS": "2:oid:0x290000000007e9,oid:0x290000000007eb",
                             "SAI_TUNNEL_ATTR_ENCAP_SRC_IP": "10.1.0.32",
                             "SAI_TUNNEL_ATTR_ENCAP_DST_IP": "11.1.0.32",
                            "SAI_TUNNEL_ATTR_ENCAP_MAPPERS": "2:oid:0x290000000007ea,oid:0x290000000007ec",
                            "SAI_TUNNEL_ATTR_TYPE": "SAI_TUNNEL_TYPE_VXLAN"
                       },
                },
               {
                   "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_TERM_TABLE_ENTRY:oid:0x2b0000000007e1": {
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_ACTION_TUNNEL_ID": "oid:0x2a0000000007e1",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_TUNNEL_TYPE": "SAI_TUNNEL_TYPE_VXLAN",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_VR_ID": "oid:0x3000000000010",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_DST_IP": "10.1.0.32",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_SRC_IP": "11.1.0.32",
                        "SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_TYPE": "SAI_TUNNEL_TERM_TABLE_ENTRY_TYPE_P2P"
                    }
                },
               {
                   "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007e9": {
                          "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VNI_TO_VLAN_ID"
                    }
                },
               {
                   "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007ec": {
                          "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VIRTUAL_ROUTER_ID_TO_VNI"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007ea": {
                          "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VLAN_ID_TO_VNI"
                    }
                },
                {
                    "ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007eb": {
                          "SAI_TUNNEL_MAP_ATTR_TYPE": "SAI_TUNNEL_MAP_TYPE_VNI_TO_VIRTUAL_ROUTER_ID"
                    }
                }
            ],
            "tables_not_found": [],
            "vidtorid": {
                "oid:0x2a0000000007e1": "oid:0x2a",
                "oid:0x290000000007ec": "oid:0x100000029",
                "oid:0x290000000007eb": "oid:0x300000029",
                "oid:0x290000000007ea": "oid:0x29",
                "oid:0x290000000007e9": "oid:0x200000029",
                "oid:0x2b0000000007e1": "oid:0x420000002b"
            }
        },
        "STATE_DB": {
            "keys": [],
            "tables_not_found": [
                "VXLAN_TUNNEL_TABLE"
            ]
        }
    }
}
```

### CODE REVIEW

File: tests/dump_tests/module_tests/evpn_test.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        expect = create_template_dict(dbs=["APPL_DB", "ASIC_DB", "STATE_DB"])
+        expect["APPL_DB"]["keys"].append("VXLAN_REMOTE_VNI_TABLE:Vlan2345:11.1.0.32")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL:oid:0x2a0000000007e1")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_TERM_TABLE_ENTRY:oid:0x2b0000000007e1")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007ea")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007e9")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007ec")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007eb")
+        expect["STATE_DB"]["keys"].append("VXLAN_TUNNEL_TABLE|EVPN_11.1.0.32")
+        ddiff = DeepDiff(sort_lists(returned), sort_lists(expect), ignore_order=True)
```
Comment (vivekreddynv) [inline tests/dump_tests/module_tests/evpn_test.py:87]: Can remove sort_lists, having ignore_order flag should work

File: tests/dump_tests/module_tests/evpn_test.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        params = {Evpn.ARG_NAME: "Vlan2345:11.1.0.33", "namespace": ""}
+        m_evpn = Evpn(match_engine)
+        returned = m_evpn.execute(params)
+        expect = create_template_dict(dbs=["APPL_DB", "ASIC_DB", "STATE_DB"])
+        expect["APPL_DB"]["keys"].append("VXLAN_REMOTE_VNI_TABLE:Vlan2345:11.1.0.33")
+        expect["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL")
+        expect["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP")
+        expect["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_TERM_TABLE_ENTRY")
+        expect["STATE_DB"]["keys"].append("VXLAN_TUNNEL_TABLE|EVPN_11.1.0.33")
+        ddiff = DeepDiff(sort_lists(returned), sort_lists(expect), ignore_order=True)
```
Comment (vivekreddynv) [inline tests/dump_tests/module_tests/evpn_test.py:103]: Can remove sort_lists, having ignore_order flag should work

File: tests/dump_tests/module_tests/vxlan_tunnel_map_test.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        """
+        params = {Vxlan_tunnel_map.ARG_NAME: "vtep_1336|map_1336_Vlan2345", "namespace": ""}
+        m_vxlan_tunnel_map = Vxlan_tunnel_map(match_engine)
+        returned = m_vxlan_tunnel_map.execute(params)
+        expect = create_template_dict(dbs=["CONFIG_DB", "APPL_DB", "ASIC_DB"])
+        expect["CONFIG_DB"]["keys"].append("VXLAN_TUNNEL_MAP|vtep_1336|map_1336_Vlan2345")
+        expect["APPL_DB"]["keys"].append("VXLAN_TUNNEL_MAP_TABLE:vtep_1336:map_1336_Vlan2345")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP_ENTRY:oid:0x3b0000000007ef")
+        expect["ASIC_DB"]["keys"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP:oid:0x290000000007e9")
+        ddiff = DeepDiff(sort_lists(returned), sort_lists(expect), ignore_order=True)
```
Comment (vivekreddynv) [inline tests/dump_tests/module_tests/vxlan_tunnel_map_test.py:83]: Can remove sort_lists, having ignore_order flag should work

File: tests/dump_tests/module_tests/vxlan_tunnel_map_test.py [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        """
+        params = {Vxlan_tunnel_map.ARG_NAME: "vtep_1336|map_1337_Vlan2346", "namespace": ""}
+        m_vxlan_tunnel_map = Vxlan_tunnel_map(match_engine)
+        returned = m_vxlan_tunnel_map.execute(params)
+        expect = create_template_dict(dbs=["CONFIG_DB", "APPL_DB", "ASIC_DB"])
+        expect["CONFIG_DB"]["keys"].append("VXLAN_TUNNEL_MAP|vtep_1336|map_1337_Vlan2346")
+        expect["APPL_DB"]["keys"].append("VXLAN_TUNNEL_MAP_TABLE:vtep_1336:map_1337_Vlan2346")
+        expect["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP_ENTRY")
+        expect["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP")
+        ddiff = DeepDiff(sort_lists(returned), sort_lists(expect), ignore_order=True)
```
Comment (vivekreddynv) [inline tests/dump_tests/module_tests/vxlan_tunnel_map_test.py:98]: Can remove sort_lists, having ignore_order flag should work

File: dump/plugins/evpn.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        else:
+            self.ret_temp[db]["tables_not_found"].extend([table])
+            return False
+
+
+    def init_evpn_appl_info(self, evpn_name):
+        req = MatchRequest(db="APPL_DB", table="VXLAN_REMOTE_VNI_TABLE", key_pattern=evpn_name, ns=self.ns)
+        ret = self.match_engine.fetch(req)
+        self.remote_ip = evpn_name.split(':')[-1]
+        self.remote_vlan = evpn_name.split(':')[0]
```
Comment (vivekreddynv) [inline dump/plugins/evpn.py:52]: Can have the lines 51 & 52 in the execute method itself

File: dump/plugins/evpn.py [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+        self.add_to_ret_template(req.table, req.db, ret["keys"], ret["error"])
+
+    def init_state_evpn_info(self, evpn_name):
+        state_key = "EVPN_" + self.remote_ip
+        print("state key " + state_key)
```
Comment (vivekreddynv) [inline dump/plugins/evpn.py:57]: use verbose_print() instead

File: dump/plugins/evpn.py [hunk 7]
```diff
@@ -0,0 +1,102 @@
+from dump.match_infra import MatchEngine, MatchRequest
+from dump.helper import create_template_dict
+from .executor import Executor
+
+
+class Evpn(Executor):
+    """
+    Debug Dump Plugin for EVPN Module
+    """
+    ARG_NAME = "Remote VNI"
```
Comment (vivekreddynv) [inline dump/plugins/evpn.py:10]: Does this have the scaling problem, which we've observed in the Route Module?

File: dump/plugins/vxlan_tunnel.py [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+    def init_asic_vxlan_tunnel_info(self):
+        if not self.src_ip:
+            self.ret_temp["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL")
+            return None
+        req = MatchRequest(db="ASIC_DB", table="ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL", field="SAI_TUNNEL_ATTR_ENCAP_SRC_IP", value=self.src_ip, ns=self.ns,
+                           return_fields=["SAI_TUNNEL_ATTR_TYPE", "SAI_TUNNEL_ATTR_ENCAP_DST_IP",
+                                          "SAI_TUNNEL_ATTR_ENCAP_MAPPERS", "SAI_TUNNEL_ATTR_DECAP_MAPPERS"])
+        ret = self.match_engine.fetch(req)
+        ret["keys"] = [x for x in ret["keys"] if ret["return_values"][x]["SAI_TUNNEL_ATTR_TYPE"] == "SAI_TUNNEL_TYPE_VXLAN"]
+        ret["keys"] = [x for x in ret["keys"] if not self.dst_ip and ret["return_values"][x]["SAI_TUNNEL_ATTR_ENCAP_DST_IP"] == ""
```
Comment (vivekreddynv) [inline dump/plugins/vxlan_tunnel.py:77]: Line seems large, i think it's better to split it.

File: dump/plugins/vxlan_tunnel.py [hunk 9]
```diff
@@ -0,0 +1,110 @@
+from dump.match_infra import MatchEngine, MatchRequest
+from dump.helper import create_template_dict
+from .executor import Executor
+
+
+class Vxlan_tunnel(Executor):
+    """
+    Debug Dump Plugin for PORT Module
```
Comment (volodymyrsamotiy) [inline dump/plugins/vxlan_tunnel.py:8]: Looks there is typo, it is not PORT Module.

File: dump/plugins/vxlan_tunnel_map.py [hunk 10]
```diff
@@ -0,0 +1,87 @@
+from dump.match_infra import MatchEngine, MatchRequest
+from dump.helper import create_template_dict
+from .executor import Executor
+
+
+class Vxlan_tunnel_map(Executor):
+    """
+    Debug Dump Plugin for PORT Module
```
Comment (volodymyrsamotiy) [inline dump/plugins/vxlan_tunnel_map.py:8]: Looks there is typo, it is not PORT Module.

File: dump/plugins/evpn.py [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+    def init_asic_evpn_info(self):
+        if not self.remote_ip:
+            self.ret_temp["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL")
+            return None
+        if self.remote_ip in self.asic_tunnel_cache:
+            ret = self.asic_tunnel_cache[self.remote_ip]
+        else:
+            req = MatchRequest(db="ASIC_DB", table="ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL", field="SAI_TUNNEL_ATTR_ENCAP_DST_IP", value=self.remote_ip, ns=self.ns,
+                               return_fields=["SAI_TUNNEL_ATTR_TYPE", "SAI_TUNNEL_ATTR_ENCAP_DST_IP",
```
Comment (vivekreddynv) [inline dump/plugins/evpn.py:71]: `"SAI_TUNNEL_ATTR_ENCAP_DST_IP"` doesn't seem to used

File: dump/plugins/evpn.py [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+    def init_asic_evpn_map_info(self):
+        if not (self.encap_mappers or self.decap_mappers):
+            self.ret_temp["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP")
+            return None
+        for key in self.encap_mappers:
+            if key in self.asic_tunnel_map_cache:
+                ret = self.asic_tunnel_map_cache[key]
+            else:
+                req = MatchRequest(db="ASIC_DB", table="ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP",
+                                  key_pattern = key, ns=self.ns)
```
Comment (vivekreddynv) [inline dump/plugins/evpn.py:107]: Should be `key_pattern=key`

File: dump/plugins/vxlan_tunnel.py [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+        ret = self.match_engine.fetch(req)
+        self.add_to_ret_template(req.table, req.db, ret["keys"], ret["error"])
+
+    def init_asic_vxlan_tunnel_map_info(self):
+        if not (self.encap_mappers or self.decap_mappers):
+            self.ret_temp["ASIC_DB"]["tables_not_found"].append("ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP")
+            return None
+        for key in self.encap_mappers:
+            req = MatchRequest(db="ASIC_DB", table="ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP",
+                              key_pattern = key, ns=self.ns)
```
Comment (vivekreddynv) [inline dump/plugins/vxlan_tunnel.py:103]: Should be `key_pattern=key`

File: dump/plugins/vxlan_tunnel.py [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+            ret = self.match_engine.fetch(req)
+            self.add_to_ret_template(req.table, req.db, ret["keys"], ret["error"])
+        for key in self.decap_mappers:
+            req = MatchRequest(db="ASIC_DB", table="ASIC_STATE:SAI_OBJECT_TYPE_TUNNEL_MAP",
+                              key_pattern = key, ns=self.ns)
```
Comment (vivekreddynv) [inline dump/plugins/vxlan_tunnel.py:108]: Should be `key_pattern=key`

File: dump/plugins/evpn.py [hunk 15]
```diff
@@ -0,0 +1,119 @@
...
+
+    def add_to_ret_template(self, table, db, keys, err):
+        if not err and keys:
+            self.ret_temp[db]["keys"].extend(keys)
+            return True
+        else:
+            self.ret_temp[db]["tables_not_found"].extend([table])
+            return False
+
+
```
Comment (vivekreddynv) [inline dump/plugins/evpn.py:51]: Can remove this empty line

**Review by vivekreddynv — CHANGES_REQUESTED** [review]

**Review by vivekreddynv — APPROVED** [review]

**Review by volodymyrsamotiy — APPROVED** [review]

**Pull request closed without merging**